src/openvic-simulation/economy/BuildingType.cpp

```diff
@@ -112,18 +112,27 @@ bool BuildingTypeManager::load_buildings_file(
 	)(root);
 	lock_building_types();
 
-	for (BuildingType const& building_type : building_types.get_items()) {
+	IndexedMap<BuildingType, ModifierEffectCache::building_type_effects_t>& building_type_effects =
+		modifier_manager.modifier_effect_cache.building_type_effects;
+
+	building_type_effects.set_keys(&get_building_types());
+
+	for (BuildingType const& building_type : get_building_types()) {
```

Comment:
Didn't you change this to hardcoded forts, naval bases and railroads?

Reply:
No, that was just the rules controlling whether they can be built. Min/max level modifiers still exist for every building type (we can restrict them if necessary in a later PR).